src/app/header/header.component.css

```diff
@@ -1,7 +1,11 @@
 .header{
   background-color: #cacaca;
+  margin: 0;
+  padding: 0;
 }
 .centerHeader{
   flex: 1;
 }
-
+.centerHeader h2{
```

Comment:
Rename class name to `headerTitle`, apply `text-align: center;` for the class, not specific element `h2`